app/connectors/exchange/ExportsDeclarationExchange.scala

```diff
@@ -94,7 +94,7 @@ object ExportsDeclarationExchange {
     dispatchLocation: Option[DispatchLocation] = None,
     additionalDeclarationType: Option[AdditionalDeclarationType] = None,
     consignmentReferences: Option[ConsignmentReferences] = None,
-    departureTransport: Option[DepartureTransport] = None,
+    departureTransportOld: Option[(Option[String], Option[DepartureTransport])] = None,
```

Comment:
Why?

We haven't had to do this for any other of these "split" stories?

Reply:
No backend speak with version1 but understood version1 and version2. Now frontend could undestood version1 and version2 and writes to backend with version2.